src/modules/powerrename/lib/Settings.cpp

```diff
@@ -240,7 +240,7 @@ void MRUListHandler::ParseJson()
                 else
                 {
                     std::vector<std::wstring> temp;
-                    for (uint32_t i = 0; i < min(jsonArray.Size(), size); ++i)
+                    for (long i = 0; i < min((long)jsonArray.Size(), size); ++i)

```

Comment:
The correct fix here is to change the type of `size`, we should never use `long` because on Windows 64-bit is just an int.

Reply:
I've used long, since DWORD, value we take from registry is unsigned long (and this was part of migration). But You're right, I'll change everything to int.

Reply:
you can use `DWORD` or unsigned int, as you prefer.